doc/image-from-registry.md

```diff
@@ -121,9 +121,8 @@ spec:
 
 To disable TLS security for a registry:
 
-Add the registry to the `cdi-insecure-registries` `ConfigMap` in the `cdi` namespace.
```

Comment:
I wonder if we should put a note about the configmap being deprecated?  Not sure I feel strongly about that though